tests/test_questions.py

```diff
@@ -0,0 +1,67 @@
+# Tests for questions
+
+import unittest
+import json
+
+from app import create_app
+from app.v1.models import Question
+
+
+class TestQuestions(unittest.TestCase):
+    """Class to test for questions"""
+
+    def setUp(self):
+        """Method to create app and set up test client"""
+        self.app = create_app("testing")
+        self.app = self.app.test_client()
+
+    def test_user_can_post_question(self):
+        """Method to test if a new question can be added"""
+        response = self.add_question()
```

Comment:
Please note that only test case methods can be defined in your test class. You can use class variables that should be defined in the `setUp` method.

Reply:
Done. Deleted non-test method from class and created class variable.